File: vs2003_cd02/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/QuickStart/aspplus/samples/vstudio/webforms/pagelets/vb/pagelet5.ascx.vb
'-----------------------------------------------------------------------
'  This file is part of the Microsoft .NET SDK Code Samples.
' 
'  Copyright (C) Microsoft Corporation.  All rights reserved.
' 
'This source code is intended only as a supplement to Microsoft
'Development Tools and/or on-line documentation.  See these other
'materials for detailed information regarding Microsoft code samples.
' 
'THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
'KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
'IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
'PARTICULAR PURPOSE.
'-----------------------------------------------------------------------

Imports System
Imports System.ComponentModel.Design
Imports System.Data
Imports System.Drawing
Imports System.Web
Imports System.Web.SessionState
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.HtmlControls
Imports Microsoft.VisualBasic

Namespace Pagelets.Vb.UserControls
    Public Class pagelet5
        Inherits System.Web.UI.UserControl

        Public BackColor As String = "white"

        Public Property UserId() As String
            Get
                Return UserName.Text
            End Get
            Set(ByVal Value As String)
                UserName.Text = Value
            End Set
        End Property

        Public Property Password() As String
            Get
                Return Pass.Text
            End Get
            Set(ByVal Value As String)
                Pass.Text = Value
            End Set
        End Property

        Public ReadOnly Property Valid() As Boolean
            Get
                Return Page.IsValid
            End Get
        End Property

#Region " Web Forms Designer Generated Code "

        Protected WithEvents UserName As System.Web.UI.WebControls.TextBox
        Protected WithEvents Pass As System.Web.UI.WebControls.TextBox
        Protected WithEvents SubmitBtn As System.Web.UI.WebControls.Button
        Protected WithEvents Validator1 As System.Web.UI.WebControls.RegularExpressionValidator
        Protected WithEvents Validator2 As System.Web.UI.WebControls.RequiredFieldValidator
        Protected WithEvents Validator3 As System.Web.UI.WebControls.RequiredFieldValidator

        Dim WithEvents pagelet5 As System.Web.UI.UserControl

        Sub New()
            pagelet5 = Me
            AddHandler pagelet5.Init, New System.EventHandler(AddressOf Me.pagelet5_Init)
        End Sub

        'CODEGEN: This procedure is required by the Web Form Designer
        'Do not modify it using the code editor.
        Private Sub InitializeComponent()
            AddHandler pagelet5.Load, New System.EventHandler(AddressOf Me.pagelet5_Load)
        End Sub

#End Region

        Protected Sub pagelet5_Load(ByVal Sender As System.Object, ByVal e As System.EventArgs)
            If Not IsPostBack Then   ' Evals true first time browser hits the page	

            End If
        End Sub

        Protected Sub pagelet5_Init(ByVal Sender As System.Object, ByVal e As System.EventArgs)
            'CODEGEN: This method call is required by the Web Form Designer
            'Do not modify it using the code editor.
            InitializeComponent()
        End Sub

    End Class
End Namespace

File: vs2003_cd02/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/QuickStart/aspplus/samples/vstudio/webforms/pagelets/cs/pagelet5.ascx.cs
//-----------------------------------------------------------------------
//  This file is part of the Microsoft .NET SDK Code Samples.
// 
//  Copyright (C) Microsoft Corporation.  All rights reserved.
// 
//This source code is intended only as a supplement to Microsoft
//Development Tools and/or on-line documentation.  See these other
//materials for detailed information regarding Microsoft code samples.
// 
//THIS CODE AND INFORMATION ARE PROVIDED AS IS WITHOUT WARRANTY OF ANY
//KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
//IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//PARTICULAR PURPOSE.
//-----------------------------------------------------------------------

namespace Pagelets.Cs.UserControls
{
    using System;
    using System.Collections;
    using System.ComponentModel;
    using System.Data;
    using System.Data.SqlClient;
    using System.Xml;
    using System.Drawing;
    using System.Web;
    using System.Web.SessionState;
    using System.Web.UI;
    using System.Web.UI.WebControls;
    using System.Web.UI.HtmlControls;

    /// <summary>
    ///    Demonstration of a Login UserControl with field validation.
    ///    Codebehind class for pagelet5.ascx.
    /// </summary>
    public class pagelet5 : System.Web.UI.UserControl
    {
        protected internal System.Web.UI.WebControls.TextBox User;
        protected internal System.Web.UI.WebControls.TextBox Pass;
        protected internal System.Web.UI.WebControls.Button SubmitBtn;
        protected internal System.Web.UI.WebControls.RegularExpressionValidator Validator1;
        protected internal System.Web.UI.WebControls.RequiredFieldValidator Validator2;
        protected internal System.Web.UI.WebControls.RequiredFieldValidator Validator3;

        public String BackColor = "white";

        public String UserId {
            get {
                return User.Text;
            }
            set {
                User.Text = value;
            }
        }

        public String Password {
            get {
                return Pass.Text;
            }
            set {
                Pass.Text = value;
            }
        }

        public bool IsValid {
            get {
                return Page.IsValid;
            }
        }

        public pagelet5() {
            this.Init += new System.EventHandler(UserControl_Init);
        }

        protected void UserControl_Load(object sender, EventArgs e)
        {

        }

        protected void UserControl_Init(object sender, EventArgs e)
        {
            //
            // CODEGEN: This call is required by the ASP.NET Windows Form Designer.
            //
            InitializeComponent();
        }   

        /// <summary>
        ///    Required method for Designer support - do not modify
        ///    the contents of this method with the code editor.
        /// </summary>
        private void InitializeComponent()
        {
	        this.Load += new System.EventHandler (this.UserControl_Load);
        }
    }
}
